content/en/apps/tutorials/application-tests.md

```diff
@@ -116,8 +134,7 @@ Let's look at a more detailed example. <a name="assessment-form-test">Here</a> i
 
 {{< highlight js "linenos=table,hl_lines=26 31 36,anchorlinenos=true,lineanchors=assessment-form-test" >}}
```

Comment:
oh cool! I didn't know about the `highlight js` feature - that's great!

otherwise, we don't use line numbers and the other items don't seem to be doing anything. we can standardize on our ` ``` ` syntax for code which I see still works with the highlight feature:

```suggestion
```js highlight 
```